.circleci/config.yml

```diff
@@ -39,6 +40,8 @@ commands:
 
   install-dependencies:
     steps:
+      - node/install:
```

Comment:
Updated to node 24.11 so that svelte + pnpm wouldn't complain during svelte-check.